### ISSUE TITLE
Syntax error in WebIDL (A.1 Interface Definition in Content Doc 3.2)

### ISSUE THREAD

**Issue by murata2makoto** [w3c/epub-specs#1071]
I used https://www.w3.org/2009/07/webidl-check? to check the syntactical correctness of this WebIDL.

I got the following error.

> The following errors were found in the extracted WebIDLs:
> 
> Extended attribute Unforgeable used on Operation — Unforgeable expected only on attributes.

It appears that the third occurrence of [Unforgeable]  causes this error.

**Comment by dauwhe** [comment]
Does anybody know WebIDL well enough to fix this? I'm afraid to touch it.

**Comment by dauwhe** [comment]
[Link to the definition in question](
http://w3c.github.io/publ-epub-revision/epub32/spec/epub-contentdocs.html#app-ers-idl)

```js
[Exposed=(Window)]
interface EpubReadingSystem {
    [Unforgeable] readonly attribute DOMString name;
    [Unforgeable] readonly attribute DOMString version;
    [Unforgeable] boolean hasFeature(DOMString feature, optional DOMString version);
};

partial interface Navigator {
    [Unforgeable, SameObject] readonly attribute EpubReadingSystem epubReadingSystem;
};
```

**Comment by dauwhe** [comment]
Fixed via #1145 after consulting with Hober.
